text-ui-test/input.txt

```diff
@@ -0,0 +1,6 @@
+todo read book
+todo return book
```

Comment:
Nice that the input.txt is working! Perhaps keeping it blank is better as users who want to try or test the application would not have such tasks already listed for them! Great job :)